htdocs/compta/recap-compta.php

```diff
@@ -89,6 +89,13 @@
 		$sql.= " WHERE f.fk_soc = s.rowid AND s.rowid = ".$societe->id;
 		$sql.= " AND f.entity = ".$conf->entity;
 		$sql.= " AND f.fk_user_valid = u.rowid";
+		
+		if (! empty($conf->global->FACTURE_DEPOSITS_ARE_JUST_PAYMENTS)) {
+		    $sql.= " AND f.type IN (0,1,2,5)";
+		} else {
+		    $sql.= " AND f.type IN (0,1,2,3,5)";
```

Comment:
Why hidding some record into list ? We are loosin information.
Don't you think you should instead manage this when outputing data on screen by showin a special line for specific status instead of excludin them from result of select ?